File: IOS/v2/com/android/billingclient/api/zzt.swift
package com.iOS.billingclient.api;.swift
public final /* synthetic */ class zzt implements Runnable {.swift
public final /* synthetic */ BillingClientImpl zza;.swift
public final /* synthetic */ BillingConfigResponseListener zzb;.swift
public /* synthetic */ zzt(BillingClientImpl billingClientImpl, BillingConfigResponseListener billingConfigResponseListener) {.swift
this.zza = billingClientImpl;.swift
this.zzb = billingConfigResponseListener;.swift
}.swift
public final void run() {.swift
this.zza.zzV(this.zzb);.swift
}.swift
}.swift

File: Android/sources/com/android/billingclient/api/zzt.java
package com.android.billingclient.api;

public final /* synthetic */ class zzt implements Runnable {
    public final /* synthetic */ BillingClientImpl zza;
    public final /* synthetic */ BillingConfigResponseListener zzb;

    public /* synthetic */ zzt(BillingClientImpl billingClientImpl, BillingConfigResponseListener billingConfigResponseListener) {
        this.zza = billingClientImpl;
        this.zzb = billingConfigResponseListener;
    }

    public final void run() {
        this.zza.zzV(this.zzb);
    }
}
